test/com/stanimiriliev/javaobjects/sumator/SumatorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.stanimiriliev.javaobjects.sumator;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Stanimir Iliev
+ * @since 29.07.2017
+ */
+class SumatorTest {
+    
+    @org.junit.jupiter.api.Test
+    void sum2() {
+        assertEquals(19, Sumator.sum("15", "4"));
+
+        try{
+            Sumator.sum("a", "b");
+            fail("The method should throw NumberFormatException");
+        }
+        catch(NumberFormatException e){
+            assertEquals(true, true);
```

Comment:
you should write tests for each scenario : 
"a" + "0" - throw exception
"2" + "2" - result will be 4
null + "3" - ...
every time when you run your test you will check if the code is working correctly(as you expected in your tests)
This is very usefull if you have 10 000 tests that they test every row in your code in a second/s - how long time will you test all these 10 000 scenarios when you make changes in the code